include/uapi/sound/sof/tokens.h

```diff
@@ -96,7 +96,7 @@
  */
 #define SOF_TKN_COMP_SINK_PIN_BINGING_WNAME	413
 #define SOF_TKN_COMP_SRC_PIN_BINGING_WNAME	414
```

Comment:
not for you @libinyang. Am I reading this correctly @aiChaoSONG ? BINGING? Did you mean BINDING?

Reply:
Oh, yeah, file a fixup, #3877